### PR TITLE
Remove GPL licensed files that are not used by the project

### PR DESCRIPTION
**Pull request by juj**

### CODE REVIEW

File: third_party/readme.txt [hunk 1]
```diff
@@ -1,2 +0,0 @@
-The contents of this folder are from GCC, and are licensed
-under the GPL (see details in the files themselves).
```
Comment (sbc100) [inline third_party/readme.txt]: I think wethird_party/websockify is still under GPL, but there is a license file in that directory so deleting this seems reasonable.

**Comment by kripken** [comment]
I think the demangler is used in a test. Can maybe move it into tests/ if that's clearer.

**Comment by sbc100** [comment]
I like the idea of a `tests/third_party` directory if we decide to move it there.

**Comment by juj** [comment]
Hmm, let me close this, I don't have cycles to revise the tests, and ended up filtering these files out in our CI distribution, which was simpler.

**Comment by sbc100** [comment]
@juj ,  BTW if you are looking for a way to build stripped down version of emscripten, I'm trying to move emsdk and other distributions over to using this "install" script which produces minimal version of emscripten:  https://github.com/emscripten-core/emscripten/blob/master/tools/install.py

**Pull request closed without merging**